### PR TITLE
Storage values of the API-Server

### PR DESCRIPTION
**Pull request by jasmingacic**

### CODE REVIEW

File: charts/api-server/templates/deployment.yaml [hunk 1]
```diff
@@ -38,6 +38,21 @@ spec:
...
           - name: TESTKUBE_TELEMETRY_DISABLED
             value: "{{ .Values.telemetryDisabled }}"
+          - name: "STORAGE_ENDPOINT"
+            value:  "{{ .Values.storage.endpoint }}"
+          - name: "STORAGE_STORAGE_ACCESSKEYID"
+            value:  "{{ .Values.storage.accessKeyId }}"
+          - name: "STORAGE_SECRETACCESSKEY"
+            value: "{{ .Values.storage.accessKey }}"
+          - name: "STORAGE_LOCATION"
+            value: "{{ .Values.storage.location }}"
```
Comment (fog1985) [inline charts/api-server/templates/deployment.yaml:48]: Hey @jasmingacic , you are calling it but it's not defined in Helm chart.

File: charts/api-server/templates/deployment.yaml [hunk 2]
```diff
@@ -38,6 +38,21 @@ spec:
...
+          - name: "STORAGE_TOKEN"
+            value: "{{ .Values.storage.token }}"
```
Comment (fog1985) [inline charts/api-server/templates/deployment.yaml:50]: The same here. It has not been defined in helm charts.

File: .github/workflows/helm-releaser.yaml [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    steps:
+      - name: Trigger GitHub pages rebuild
+        run: |
+          curl --fail --request POST \
+            --url https://api.github.com/repos/kubeshop/helm-charts/pages/builds \
+            --header "Authorization: Bearer $USER_TOKEN"
+        env:
+          # You must create a personal token with repo access as GitHub does
+          # not yet support server-to-server page builds.
+          USER_TOKEN: ${{ secrets.USER_TOKEN }}
```
Comment (fog1985) [inline .github/workflows/helm-releaser.yaml:188]: @jasmingacic , the whole this file seems to be obsolete. Not sure where did you get it. Please get a new one from the main branch. It's 296 lines long. And has much more blocks of code and logic.

**Review by fog1985 — CHANGES_REQUESTED** [review]
Please address those comments.

File: .github/workflows/helm-releaser.yaml (hunk 3 above)
Comment (jasmingacic) [reply .github/workflows/helm-releaser.yaml:188]: this was not supposed to be in this PR no clue how i got it

**Pull request closed without merging**